src/groups/mqb/mqbmock/mqbmock_queue.cpp

```diff
@@ -528,7 +528,8 @@ mqbi::QueueHandle* HandleFactory::makeHandle(
     const bsl::shared_ptr<mqbi::QueueHandleRequesterContext>& clientContext,
     mqbstat::QueueStatsDomain*                                stats,
     const bmqp_ctrlmsg::QueueHandleParameters&                handleParameters,
-    bslma::Allocator*                                         allocator)
+    BSLS_ANNOTATION_UNUSED mqbu::SingleCounter* parent,
```

Comment:
I am surprised to see the formatter misaligning the arguments list here.
Not needed to fix, but it might cause reformatting when we update clang-format